test/unit/commands-test.js

```diff
@@ -57,7 +57,7 @@ describe('Command attributes test', function() {
     });
     afterEach(function(done) {
         async.series([
-            apply(lwm2mClient.unregister, deviceInformation),
+            /*apply(lwm2mClient.unregister, deviceInformation),*/
```

Comment:
Leftover comment?

Reply:
Fixed in c86cf5c10739653161112be9133ff978bb8c386c